src/app/component/in-sight-side-bar/in-sight-side-bar.component.html

```diff
@@ -4,10 +4,87 @@
+        <div class="assessments-wrapper">
+            <div class="assessments-info-wrapper">
+                <div class="flex justify-between">
+                    <div class="flex flex-col">
+                        <div class="flex flex-between flex-middle">
+                            <p class="flex-align-start upcoming-assessment-title flex flex-middle margin-remove">Upcoming Assessment
+                                <img class="margin-left-s cursor-pointer" width="20px" height="20px"
+                                    src="assets/icons/home/help_icon.svg" #tooltip="matTooltip" [matTooltipClass]="'below'"
+                                    matTooltip="View your Credentials and take your Assessment.">
+                            </p>
+                        </div>
+                        <button
+                            class="bg-blue-700 text-sm font-semibold text-white py-3 px-5 rounded-full border-none cursor-pointer mt-4 mb-2 tracking-wide"
+                            [routerLink]="['/app/discussion-forum']" >START</button>
+                    </div>
+                    <div class="flex justify-center system-icon">
+                        <mat-icon class="custom-size">desktop_mac</mat-icon>
+                    </div>
+                </div>
+            </div>
+            <div class="cred-container">
+                <div class="flex flex-middle flex-between">
+                    <div class="flex flex-middle">
+                        <span class="cred-title">{{ credMessage }}</span>
+                    </div>
+                    <div class="flex">
+                        <mat-icon class="ws-mat-default-text cursor-pointer" (click)="toggleCreds()">{{showCreds ? 'keyboard_arrow_up' : 'keyboard_arrow_down'}}</mat-icon>
+                    </div>
+                </div>
+                <div class="my-credentials p-3" *ngIf="showCreds">
+                    <div class="flex flex-middle" style="padding: 5px 0px;">
+                        <span class="pr-5 font-semibold">Username</span>
+                        <div class="flex flex-middle copy-items">
+                            <span class="pr-2 custom-width">{{assessmentsData.userName}}</span>
+                            <span class=""><mat-icon class="cursor-pointer copy-icon" (click)="copyToClipboard('Venkat')">content_copy</mat-icon></span>
+                        </div>
+                    </div>
+                    <div class="flex flex-middle" style="padding: 5px 0px;">
+                        <span class="pr-6 font-semibold">Password</span>
+                        <div class="flex flex-middle copy-items">
+                            <span class="pr-2  custom-width">{{assessmentsData.password}}</span>
+                            <span class=""><mat-icon class="cursor-pointer copy-icon" (click)="copyToClipboard('Test@1234')">content_copy</mat-icon></span>
```

Comment:
are we showing user name and password here?

Reply:
We commented on this while enabling I can do the transition.